common/test/resources/reference-navigation.json

```diff
@@ -1127,7 +1127,7 @@
 				},
 				{
 					"title": "Soccer",
-					"url": "/soccer",
+					"url": "/us/soccer",
```

Comment:
Is this file used by a test somewhere? If so should we update the test to assert that these are the links for the US edition?